sqlmesh/dbt/converter/convert.py

```diff
@@ -219,12 +221,18 @@ def _convert_models(
             assert isinstance(model.kind, SeedKind)
             model.kind.path = str(Path("../seeds", seed_filename))
         else:
+            if model._path is None:
+                raise ValueError(f"Unhandled model path: {model._path}")
```

Comment:
Should this also contain the model's name? This just interpolates `None` into the message.